actors/miner/src/expiration_queue.rs

```diff
@@ -35,8 +36,11 @@ const ENTRY_SECTORS_MAX: u64 = 10_000;
 pub struct ExpirationSet {
     /// Sectors expiring "on time" at the end of their committed life
```

Comment:
Now that we're calling early_sectors faulty we should call out in comment that on_time_sectors may also be faulty